### PR TITLE
SPA vs web

### PR DESCRIPTION
**Pull request by albertchen-okta**
## Description:
- **What's changed?** 
- making it more clear why devs should choose SPA vs Web
-
- **Is this PR related to a Monolith release?** 
no

### Resolves:

recent trend of devs not knowing which one to pick and picking the wrong one

### CODE REVIEW

**Comment by swiftone** [comment]
Nice - could we update https://github.com/okta/okta-developer-docs/blob/albertchen-okta-patch-2/packages/%40okta/vuepress-site/docs/guides/quickstart/register-app/website/register-app.md with the same text?   It currently says just to pick "Web".

**Comment by albertchen-okta** [comment]
done, thanks

**Review by swiftone — APPROVED** [review]

File: packages/@okta/vuepress-site/docs/guides/quickstart/register-app/cli/register-app.md [hunk 1]
```diff
@@ -2,6 +2,6 @@ To use the Okta CLI:
 
 1. Enter the command: `okta apps create`.
 2. Give your app a name.
-3. Select **SPA** (for Angular, React or Vue) or **Web** as the  **Type of Application**.
+3. Select **SPA** or **Web** as the **Type of Application**. Select **SPA** if your application will be using the tokens in Angular, React, Vue, or other browser-side code.  Pick **Web** if your application will be using the tokens solely in server-side code.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/quickstart/register-app/cli/register-app.md:5]:
```suggestion
3. Select **SPA** or **Web** as the **Type of Application**. Select **SPA** if your application is using the tokens in Angular, React, Vue, or other browser-side code.  Pick **Web** if your application is using the tokens solely in server-side code.
```

File: packages/@okta/vuepress-site/docs/guides/quickstart/register-app/website/register-app.md [hunk 2]
```diff
@@ -4,7 +4,7 @@ In the Admin Console, complete these steps:
 
 2. Click **Add Application**.
 
-3. Select **Web** as the type of application. Click **Next**.
+3. Select **SPA** or **Web** as the **Type of Application**. Select **SPA** if your application will be using the tokens in Angular, React, Vue, or other browser-side code. Pick **Web** if your application will be using the tokens solely in server-side code.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/quickstart/register-app/website/register-app.md:7]:
```suggestion
3. Select **SPA** or **Web** as the **Type of Application**. Select **SPA** if your application is using the tokens in Angular, React, Vue, or other browser-side code. Pick **Web** if your application is using the tokens solely in server-side code.
```

**Review by susanharper-okta — APPROVED** [review]
future -> active tense - no biggie.

**Comment by swiftone** [comment]
Passive voice will ever be the nemesis that will bring me down.

**Pull request merged**